back/src/models/reactionData.model.ts

```diff
@@ -1,41 +1,46 @@
 // reactionData.model.ts
 
-import { Sequelize, DataTypes, Model } from 'sequelize';
+import {Sequelize, DataTypes, Model} from 'sequelize';
 
 class ReactionData extends Model {
+    public id!: number;
+    public ownerId!: number;
+    public data!: JSON;
+    public reactionId!: number;
+    public oauthId!: number;
 
-  public static initialize(sequelize: Sequelize) {
-    ReactionData.init(
-      {
-        id: {
-          type: DataTypes.INTEGER,
-          primaryKey: true,
-          allowNull: false,
-        },
-        owner_id: {
-          type: DataTypes.INTEGER,
-          allowNull: false,
-        },
-        data: {
-          type: DataTypes.JSON,
-          allowNull: true,
-        
-        },
-        actionsDataIds: {
-          type: DataTypes.ARRAY(DataTypes.INTEGER),
-          allowNull: true,
-        },
-        oauthId : {
-          type: DataTypes.INTEGER,
-          allowNull: true,
-        },
-      },
-      {
-        sequelize,
-        tableName: 'reactionData',
-      }
-    );
-  }
+    public static initialize(sequelize: Sequelize) {
+        ReactionData.init(
+            {
+                id: {
+                    type: DataTypes.INTEGER,
+                    primaryKey: true,
+                    allowNull: false,
+                },
+                ownerId: {
+                    type: DataTypes.INTEGER,
+                    allowNull: false,
+                },
+                data: {
+                    type: DataTypes.JSON,
+                    allowNull: true,
+
+                },
```

Comment:
There should be no empty line here
```suggestion
                    allowNull: true,
                },
```